### PR TITLE
Ci fix

### PR DESCRIPTION
**Pull request by David-OConnor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,8 @@
 
 https://seed-rs.org/
 
-Based on https://github.com/seed-rs/seed-quickstart-webpack
+Based on [Seed-quickstart-webpack](https://github.com/seed-rs/seed-quickstart-webpack]).
+
+To build this site in a local environment, follow the instructions in the above link.
+Make sure to use `rustc nightly` - run `rustup default nightly`, in order for the verification
+steps to pass.
```
Comment (MartinKavik) [inline README.md]: Are you sure users need to switch to `nightly`?

**Comment by David-OConnor** [comment]
No - This appears to be wrong; was able to get verify to pass on stable now.

**Comment by MartinKavik** [comment]
Nightly has to be installed (maybe `cargo-make` can install it automatically), but you don't have to switch it because it's selected explicitly in `Makefile.toml`:
```toml
[tasks.fmt]
description = "Format with nightly rustfmt"
toolchain = "nightly"
command = "cargo"
args = ["fmt", "--all"]
```

**Comment by David-OConnor** [comment]
That makes sense; I didn't have it set up before, and the fail didn't provide any details. (I can't remember the message, but it wasn't the usual failed test etc).

**Comment by MartinKavik** [comment]
Yeah it's possible - for instance I had some weird problems with compilation / verification when my RAM was exhausted.

**Pull request merged**